06.bowling_object/game.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+class Game
+  def initialize(shots)
+    game = []
+    shots.each do |shot|
+      game << Shot.new(shot)
```

Comment:
```suggestion
    shots = []
    shots.each do |shot|
      shots << Shot.new(shot)
```

Shotを配列に入れるならshotsが適切です。
そうすると引数のshotsと名前がかぶりますが、異なるものに同じ名前を付けていることになるので、どちらかを適切な名前に変える必要があります。

また、ここはeachではなくmapが使えますね。